upgrade/Upgrade-1.10.3.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @return bool
+ */
+function upgrade_module_1_10_0($module)
```

Comment:
@synicko wrong version number for this module method, should be 1_10_3 ;)

Reply:
Thanks Jocelyn, I make the fix tight now